ipfs/library/Network/IPFS/Client/Files/Write/Form/Types.hs

```diff
@@ -0,0 +1,23 @@
+module Network.IPFS.Client.Files.Write.Form.Types (Form (..)) where
+
+import           RIO
+import qualified RIO.ByteString.Lazy     as Lazy
+
+import           Servant.Multipart
+import           Servant.Multipart.API
+
+import qualified Network.IPFS.File.Types as File
+
+data Form = Form
```

Comment:
Since this only has one field, this could also be a `newtype`

Reply:
(Which is really just a performance optimization. The compiler can't automatically force this because of how it interacts with laziness — newtypes are always strict)